controllers/pod_node_module_reconciler.go

```diff
@@ -94,6 +116,9 @@ func (pnmr *PodNodeModuleReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if err := pnmr.addLabel(ctx, nodeName, labelName); err != nil {
 		return ctrl.Result{}, fmt.Errorf("could not label node %s with %q: %v", nodeName, labelName, err)
 	}
+	if err := pnmr.addLabel(ctx, nodeName, deprecatedLabelName); err != nil {
+		return ctrl.Result{}, fmt.Errorf("could not label node %s with %q: %v", nodeName, deprecatedLabelName, err)
+	}
```

Comment:
Same comment about two patch operations where just one would be preferable.

Reply:
Fixed.